tests/unit/llms/providers/test_openai.py

```diff
@@ -26,7 +27,28 @@ def test_basic_interface():
         }
     )
 
-    openai = OpenAI(model="uber-model", params={}, provider_config=provider_cfg)
+
+def test_basic_interface(provider_config):
+    """Test basic interface."""
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    openai = OpenAI(model="uber-model", params={}, provider_config=provider_config)
+    llm = openai.load()
+    assert isinstance(llm, ChatOpenAI)
+    assert openai.default_params
+
+
+def test_params_handling(provider_config):
+    """Test that now allowed parameters are removed before model init."""
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    # these two parameters should be removed before model init
+    params = {
+        "min_new_tokens": 1,
+        "max_new_tokens": 10,
+    }
+
+    openai = OpenAI(model="uber-model", params=params, provider_config=provider_config)
```

Comment:
where are you actually checking that the disallowed params were removed?

Reply:
right! added some checks